src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -254,8 +254,20 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                         var source = remoteProvider.Source;
                         if (source.IsHttp && !source.IsHttps && !source.AllowInsecureConnections)
                         {
-                            await _logger.LogAsync(RestoreLogMessage.CreateError(NuGetLogCode.NU1302,
+                            var isErrorEnabled = SdkAnalysisLevelMinimums.IsEnabled(_request.Project.RestoreMetadata.SdkAnalysisLevel,
+                                _request.Project.RestoreMetadata.UsingMicrosoftNETSdk,
+                                SdkAnalysisLevelMinimums.HttpErrorSdkAnalysisLevelMinimumValue);
```

Comment:
We can see here how "wordy" the property name is. The C# language doesn't let you (easily) use the property without also specifying the class name, and having "SdkAnalysisLevelMinimum" in both the class and property looks redundant when used.

But this is something easily changed later, since it's not a public API.  Just something to keep in mind when writing new code.